tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.html

```diff
@@ -339,14 +345,19 @@
             datum.run;
       },
       _getColorScale() {
-        return {
-          scale: (name) => {
-            const [exp, run] = JSON.parse(name);
-            return this.multiExperiments ?
-                tf_color_scale.experimentsColorScale(exp) :
-                tf_color_scale.runsColorScale(run);
-          },
-        };
+        // If 'colorScale' isn't explicitly specified, use the ones
+        // defined in tf_color_scale.
+        if (this.colorScale === null) {
```

Comment:
code would format tiny bit nicer if you early terminate when `this.colorScale` exists.

Reply:
Done.